app/views/layouts/hyacinth_v1.html.erb

```diff
@@ -10,7 +10,7 @@
     <% # stylesheet_pack_tag is available as long as javascript_pack_tag includes at least one css file %>
     <%= stylesheet_pack_tag 'hyacinth_v1' %>
   </head>
-  <body class="hyacinth-ui-v1">
+  <body class="hyacinth-ui-v1" data-hyacinth-version="<%= File.read(File.join(Rails.root, '/VERSION')).strip %>">
```

Comment:
Future enhancement - it would be good to cache this on startup to avoid the extra disk i/o on every render.

Reply:
Good idea.  I'll do that now actually.